src/translations/index.ts

```diff
@@ -419,4 +432,34 @@ export namespace TranslationsModel {
     export interface ListProjectBuildsOptions extends PaginationOptions {
         branchId?: number;
     }
+
+    export interface PreTranslationReport {
+        languages: TargetLanguage[];
+        preTranslateType: string;
```

Comment:
I'd suggest using the `Method` type here. For some reason, the terminology is different here, but it's the same set of values.